core/chains/evm/logpoller/models.go

```diff
@@ -0,0 +1,32 @@
+package logpoller
+
+import (
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/lib/pq"
+
+	"github.com/smartcontractkit/chainlink/core/utils"
+)
+
+type LogPollerBlock struct {
+	EvmChainId *utils.Big
+	BlockHash  common.Hash
+	// TODO: Following existing data types for block numbers, though not sure why we don't use big.Ints elsewhere?
```

Comment:
question for @samsondav

Reply:
we do have to update the db data type, bigint corresponds to int64. Sounds good though, we can make this service the first to actually use big ints (and corresponding numeric types in the db)

Reply:
on second thought, moving away from int64 is fairly painful because the big.Int API is so unergonomic. Internally geth uses an int64 anyways https://github.com/ethereum/go-ethereum/blob/f66f1a16b3c480d3a43ac7e8a09ab3e362e96ae4/eth/filters/api.go#L340 and even at a block per microsecond it'd take a 30k years to run out so I think its ok.